api/ruby/ghe-org-permissions-report.rb

```diff
@@ -0,0 +1,119 @@
+#!/usr/bin/env ruby
+# Generates a CSV report listing all organizations, their repositories,
+# collaborators, effective permissions, teams, and team permissions
+#
+# Set OCTOKIT_ACCESS_TOKEN to a token with read:org scope owned by a site admin
+# and OCTOKIT_API_ENDPOINT to http(s)://[your-hostname]/api/v3/
+#
+# Use `ghe-org-admin-promote` to make a site admin an owner of all
+# organizations
+require 'octokit'
+
+Octokit.default_media_type = 'application/vnd.github.ironman-preview+json'
+ghe = Octokit::Client.new
```

Comment:
[It's an Octokit.rb default](https://github.com/octokit/octokit.rb/blob/28a594ff371ff3faf8ec50dd25bd65bd779ff43f/lib/octokit/default.rb#L45)